packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -113,6 +113,13 @@ export class Slider extends FormAssociated<HTMLInputElement>
     @observable
     public trackMinHeight: number = 0;
 
+    /**
+     * Custom function that generates a string for the component's "aria-valuetext" attribute based on the currrent value.
+     *
+     * @public
+     */
+    @observable valueTextFormatter: (value: string) => string | null = () => null;
```

Comment:
nit, can you denote this as `public` in TS as well?

```suggestion
@observable
public valueTextFormatter: (value: string) => string | null = () => null;
```

Reply:
updated